README.md

```diff
@@ -532,6 +534,47 @@ let mainItems = try myContext.execute(request)
 let relatedItem = myMainItem.manyToMany.first
 ```
 
+## Files
+
+```yaml
+files:
+  inputs: path/to/search
+  filter: .+\.xib$
```

Comment:
I'm wondering if this doesn't risk to be a confusing example, using `.xib` as the filter here. Maybe using `.+\.txt$` or `.+\.mp4$` in that example would avoid the risk of confusion with XIBs and the `ib` parser?